1-Authentication/7-sign-in-express-mfa/Api/authConfig.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+require("dotenv").config({ path: ".env.dev" });
+
+const TENANT_SUBDOMAIN =
+  process.env.TENANT_SUBDOMAIN || "Enter_the_Tenant_Subdomain_Here";
+const TENANT_ID = process.env.TENANT_ID || "Enter_the_Tenant_Subdomain_Here";
+const REDIRECT_URI =
+  process.env.REDIRECT_URI || "http://localhost:3000/auth/redirect";
+const POST_LOGOUT_REDIRECT_URI =
+  process.env.POST_LOGOUT_REDIRECT_URI || "http://localhost:3000";
+
+/**
+ * Configuration object to be passed to MSAL instance on creation.
+ * For a full list of MSAL Node configuration parameters, visit:
+ * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/configuration.md
+ */
+const msalConfig = {
+  auth: {
+    clientId: process.env.CLIENT_ID || "Enter_the_Application_Id_Here", // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
```

Comment:
```suggestion
    clientId: process.env.CLIENT_ID || "Enter_the_Edit_Profile_Service_Application_Id_Here", // 'Application (client) ID' of the Edit_Profile Service App registration in Microsoft Entra admin center - this value is a GUID
```

Reply:
@4gust  this is the app ID of the API app. Yes?

Reply:
Yes